IMuaythai.Api/Validators/Institutions/InstitutionResponseValidator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using FluentValidation;
+using IMuaythai.Models.Institutions;
+
+namespace IMuaythai.Api.Validators.Institutions
+{
+    public class InstitutionResponseValidator : AbstractValidator<InstitutionResponseModel>
```

Comment:
I am afraid we do not need validator for this class because as class name suggests that class is used for response only. So let's remove this validator

Reply:
So ContestResponseValidator should also be deleted?

Reply:
no no, if a class name has *Response suffix then this class does not need validator